src/main/resources/templates/course_settings.html

```diff
@@ -53,8 +53,8 @@ <h6 class="align-middle my-0">Nazwa kursu:</h6>
 </section>
 
 <section class="container py-3 border-bottom"
-         th:unless="${#strings.contains(#strings.setJoin(user.getRoles(), ','), 'STUDENT')}">
-    <form method="get" th:action="|/course_settings/description/${course.getId()}|">
+         th:unless="${user.getRoles().contains(roles.STUDENT)}">
```

Comment:
this needs to be entirely reworked anyway - in our app a lecturer can be also a student, so we need to take care of that (but for now this refactoring is ok)

Reply:
Good point